pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1095,7 +1095,7 @@ func (dm *DockerManager) pluginDisablesDockerNetworking() bool {
 
 // newDockerVersion returns a semantically versioned docker version value
```

Comment:
you are right! it is used in Version function, I'll leave this function as is and lemme modify the description. We will add newDockerAPIVersion when somebody really need it!